CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 0.23.1 / TBD
 ===================
+- Install `greenkeeper-lockfile` with `npm` rather than `yarn` as latest version of `yarn` isn't working see https://github.com/greenkeeperio/greenkeeper-lockfile/issues/98
```

Comment:
This entry shouldn't be here any more, can you remove please?

Reply:
Well spotted. Updated.

Reply:
This line is still in the file, can it be deleted please?

Reply:
Sorry my comment was for the ie styles.